hooks/pre/30-upgrade.rb

```diff
@@ -74,12 +74,6 @@ def remove_nodes_distributors
   Kafo::Helpers.execute("mongo pulp_database --eval  'db.repo_distributors.remove({'distributor_type_id': \"nodes_http_distributor\"});'")
 end
 
-def fix_pulp_httpd_conf
-  return true unless File.exist?('/etc/httpd/conf.d/pulp.conf.rpmnew')
-
-  Kafo::Helpers.execute('cp /etc/httpd/conf.d/pulp.conf.rpmnew /etc/httpd/conf.d/pulp.conf')
```

Comment:
@ehelms @jlsherrill Do you know why we needed this and was marked an always-run?  It's causing problems now, the BZ has a full discussion of why, but pulp.conf now duplicates stuff pulp module creates so apache won't start.